test/test_quantization.py

```diff
@@ -90,7 +90,9 @@
     from quantization.pt2e.test_xnnpack_quantizer import TestXNNPACKQuantizer  # noqa: F401
     from quantization.pt2e.test_xnnpack_quantizer import TestXNNPACKQuantizerModels  # noqa: F401
     from quantization.pt2e.test_x86inductor_quantizer import TestQuantizePT2EX86Inductor  # noqa: F401
-    from quantization.pt2e.test_quantize_pt2e_qat import TestQuantizePT2EQAT  # noqa: F401
+    # TODO: enable this in the next PR
+    # from quantization.pt2e.test_quantize_pt2e_qat import TestQuantizePT2EQAT_ConvBn1d  # noqa: F401
+    from quantization.pt2e.test_quantize_pt2e_qat import TestQuantizePT2EQAT_ConvBn2d  # noqa: F401
```

Comment:
I see. `TestQuantizePT2EQAT_ConvBn2d` is actually a group of tests, not a single test. I just felt each pattern already has a ton of tests so it's better to break them down instead of putting them all in one giant TestCase. Do you think we should merge them instead?

Reply:
I think we have multiple ways to structure tests already so ideally want to keep it simpler, we already have different folders, test files, then TestCases and then test names

Reply:
Ok, added a TODO. As discussed offline we will merge these test cases into a single one in the future